Buildings/Buildings.csv

```diff
@@ -199,7 +199,6 @@ building_id,building_code,parent_code,building_name,alternate_building_names,lat
 149,KSW,,195 King Street West,,,,1
 150,HSC,,Huntsville Summit Centre,,45.32211,-79.20695,1
 151,UAE,,United Arab Emirates Dubai,,,,1
-152,WSS,,"6 Wellington Street, Stratford",,43.370474,-80.982231,1
```

Comment:
Hi Kartik,

WSS = It was a temporary location while DMS was being built.
KSW = I think the address is wrong, it should be “180 King St. S., Suite 700” (source: https://uwaterloo.ca/master-of-public-service/)
HSC =  “Huntsville Summit Centre” should be “Waterloo Summit Centre for the Environment” (source: https://uwaterloo.ca/waterloo-summit-centre/about-waterloo-summit-centre/contact-us)
UAE = Should be closed (see http://www.therecord.com/news-story/2615152-university-of-waterloo-closing-its-dubai-campus/). However, their website is still up and running: https://info.uwaterloo.ca/uae/index.php.

Thanks,

Michael Tjendra
Client Support Specialist
Information Systems & Technology
University of Waterloo
519-888-4567, ext. 45857

From: Kartik Talwar [mailto:notifications@github.com]
Sent: Wednesday, May 13, 2015 4:04 PM
To: uWaterloo/Datasets
Cc: Michael Tjendra
Subject: Re: [Datasets] Update Buildings.csv (#114)


In Buildings/Buildings.csv<https://github.com/uWaterloo/Datasets/pull/114#discussion_r30269881>:

> @@ -199,7 +199,6 @@ building_id,building_code,parent_code,building_name,alternate_building_names,lat

>  149,KSW,,195 King Street West,,,,1

>  150,HSC,,Huntsville Summit Centre,,45.32211,-79.20695,1

>  151,UAE,,United Arab Emirates Dubai,,,,1

> -152,WSS,,"6 Wellington Street, Stratford",,43.370474,-80.982231,1

is this not a building anymore?

—
Reply to this email directly or view it on GitHub<https://github.com/uWaterloo/Datasets/pull/114/files#r30269881>.